int32 controlword
int32 opmode
int32 target_position
